assets/scss/components/_all.scss

```diff
@@ -23,3 +23,50 @@
 @import "tabs";
 @import "togglePill";
 @import "dateTimePicker";
+
+// MOVE THIS TO SWARM-SASSTOOLS BEFORE PR
```

Comment:
Let's not merge until:

1. this swarm-sasstools PR is merged and a new release is published: https://github.com/meetup/swarm-sasstools/pull/14
2. this branch updates swarm-sasstools
3. this CSS is removed